src/path/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Path(models.Model):
+    start_location_id = models.TextField()
```

Comment:
wouldn't this make sense as an integer? also what does the id refer to?

Reply:
frontend is using google places api and id is a string: https://developers.google.com/maps/documentation/places/web-service/place-id 

will add comments to make this clearer :) tyyy